sw/rust/ibex-demo-system-hal/src/spi.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use embedded_hal::blocking;
+use embedded_hal::digital::v2::OutputPin;
+use embedded_hal::spi;
+
+use ibex_demo_system_pac::SPI0;
+
+pub struct Spi<T: OutputPin> {
+    device: SPI0,
+    chip_enable: T,
```

Comment:
This is the chip select pin (active low), right? Calling it `chip_enable`, at least to me, has a connotation that it is active high.